src/views/pages/Home/index.vue

```diff
@@ -15,6 +15,7 @@
         </md-card>
       </router-link>
 
+    <router-link :to="{ path: '/analytics' }">
```

Comment:
could you use the format `:to="{ name: '<name of the route>' }"? this is just so that we don't have to hardcode our paths everywhere.

also, could you add the route to the tabs? should be in the toolbar component!

Reply:
oky doky